src/client/method-status.c

```diff
@@ -21,6 +21,13 @@ struct bus_properties_map {
         size_t offset;
```

Comment:
Not part of this PR, but would be great to fix it here:

In [src/client/client.c#115](https://github.com/eclipse-bluechi/bluechi/blob/main/src/client/client.c#L115), a call to `assemble_object_path_string` is being made and the result is assigned to `client->object_path`. This causes a memory leak when called more than once since `client->object_path` doesn't get freed before. So adding the following line there (in a separate commit) would fix that issue:
```C
// src/client/client.c#114-115 (client_create_message_new_method_call)
free_and_null(client->object_path)
r = assemble_object_path_string(NODE_OBJECT_PATH_PREFIX, node_name, &client->object_path);
```